### PR TITLE
add upgrade.sh (#1365)

### PR DESCRIPTION
**Pull request by bingryan**
Upgrading the substrate branch through a script.

### CODE REVIEW

File: scripts/upgrade.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+FROM=`grep "^substrate-build-script-utils" ./node/parallel/Cargo.toml | egrep -o "(v[0-9\.]+)"`
+TO=${TO}
+
+cargo_toml_list=$(find . -name "Cargo.toml")
+echo "upgrade substrate from polkadot-${FROM} to polkadot-${TO}"
+for cargo_toml in $cargo_toml_list
+do
+    sed -i "" "s/$FROM/$TO/g" ${cargo_toml}
```
Comment (GopherJ) [inline scripts/upgrade.sh:17]: can you add linux support?

File: scripts/upgrade.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+# The following line ensure we run from the project root
+PROJECT_ROOT=`git rev-parse --show-toplevel`
+cd ${PROJECT_ROOT}
+
+
+FROM=`grep "^substrate-build-script-utils" ./node/parallel/Cargo.toml | egrep -o "(v[0-9\.]+)"`
+TO=${TO}
+
+cargo_toml_list=$(find . -name "Cargo.toml")
```
Comment (GopherJ) [inline scripts/upgrade.sh:13]: should also ignore target directory

Comment (bingryan) [reply scripts/upgrade.sh:13]: yes, target dir and Cargo.lock should ignore

**Review by GopherJ — APPROVED** [review]

**Comment by GopherJ** [comment]
LGTM thanks

**Pull request merged**